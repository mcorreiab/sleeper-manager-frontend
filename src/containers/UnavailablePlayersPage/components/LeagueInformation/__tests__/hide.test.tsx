import renderer from "react-test-renderer";
import Hide from "../hide";

it("should render expand svg without props", () => {
  expect(renderer.create(<Hide />).toJSON()).toMatchSnapshot();
});
