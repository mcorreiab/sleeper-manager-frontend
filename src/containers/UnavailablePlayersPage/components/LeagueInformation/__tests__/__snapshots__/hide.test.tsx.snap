// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expand svg without props 1`] = `
<svg
  aria-hidden="true"
  fill="none"
  height="8"
  viewBox="0 0 12 8"
  width="12"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M5.29501 1L0.705006 5.59C0.315006 5.98 0.315006 6.61 0.705006 7C1.09501 7.39 1.72501 7.39 2.11501 7L6.00501 3.12L9.88501 7C10.275 7.39 10.905 7.39 11.295 7C11.685 6.61 11.685 5.98 11.295 5.59L6.70501 1C6.32501 0.610004 5.68501 0.610004 5.29501 1Z"
    fill="#EAEAEA"
  />
</svg>
`;
